# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the 'addonPackager' package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
msgid ""
msgstr ""
"Project-Id-Version: 'addonPackager' '1.0'\n"
"Report-Msgid-Bugs-To: 'nvda-translations@groups.io'\n"
"POT-Creation-Date: 2022-06-20 13:50+0200\n"
"PO-Revision-Date: 2022-06-21 23:24+0200\n"
"Last-Translator: Ângelo Abrantes <ampa4374@gmail.com>\n"
"Language-Team: Equipa portuguesa do NVDA: Ângelo Abrantes e Rui fontes "
"(10-06-2022)\n"
"Language: pt_PT\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"
"X-Generator: Poedit 3.0\n"
"X-Poedit-SourceCharset: UTF-8\n"
"X-Poedit-Basepath: ../../../globalPlugins\n"
"X-Poedit-SearchPath-0: .\n"

#. Add-on summary, usually the user visible name of the addon.
#. Translators: Summary for this add-on to be shown on installation and add-on information.
#: buildVars.py:23 addon\globalPlugins\addonPackager\__init__.py:66
#: addon\globalPlugins\addonPackager\main.py:23
msgid "Utilidades para los complementos de NVDA"
msgstr "Utilitários para extras do NVDA"

#. Add-on description
#. Translators: Long description to be shown for this add-on on add-on information from add-ons manager
#: buildVars.py:26
msgid ""
"Conjunto de utilidades para un manejo fácil de complementos para NVDA.\n"
"\n"
"* Empaquetador de complementos.\n"
"* Instalador múltiple de complementos.\n"
"* Desinstala complementos.\n"
"* Habilita / deshabilita complementos.\n"
"* Modificador de manifiestos, instalados y desde archivo no instalado.\n"
"* Documentación de complementos.\n"
"\n"
"Todas las áreas de fácil manejo y para ahorrar tiempo respecto al gestor de "
"complementos nativo de NVDA."
msgstr ""
"Conjunto de utilitários para facilitar a gestão de add-ons NVDA.\n"
"\n"
"* Embalagem adicional.\n"
"* Instalador de múltiplos add-on.\n"
"* Desinstalar add-ons.\n"
"* Activar/desactivar add-ons.\n"
"* Modificador manifesto, instalado e a partir de ficheiro não-instalado.\n"
"* Documentação adicional.\n"
"\n"
"Todas as áreas são de fácil utilização e poupam tempo em comparação com o "
"gestor de add-on nativo da NVDA."

#: addon\globalPlugins\addonPackager\__init__.py:34
msgid "&Utilidades para los complementos de NVDA"
msgstr "&Utilitários para extras do NVDA"

#: addon\globalPlugins\addonPackager\__init__.py:51
msgid ""
"Hay una acción pendiente de reiniciar NVDA.\n"
"\n"
"Por motivos de seguridad el complemento no podrá ser usado hasta que no "
"reinicie.\n"
"\n"
"¿Desea reiniciar NVDA ahora?"
msgstr ""
"Há uma acção pendente para reiniciar o NVDA.\n"
"\n"
"Por razões de segurança, o extra não pode ser utilizado até ser reiniciado.\n"
"\n"
"quer reiniciar o NVDA agora?"

#: addon\globalPlugins\addonPackager\__init__.py:56
msgid "Pregunta"
msgstr "Pergunta"

#: addon\globalPlugins\addonPackager\__init__.py:66
msgid "Muestra la ventana de Utilidades para los complementos de NVDA"
msgstr "Apresenta a janela de Utilitários para os add-ons NVDA"

#: addon\globalPlugins\addonPackager\__init__.py:76
#: addon\globalPlugins\addonPackager\__init__.py:78
msgid ""
"Ya hay una instancia de Utilidades para los complementos de NVDA abierta."
msgstr "Já existe um exemplo de Utilitários para NVDA add-ons abertos."

#: addon\globalPlugins\addonPackager\__init__.py:76
#: addon\globalPlugins\addonPackager\main.py:354
msgid "Información"
msgstr "Informação"

#: addon\globalPlugins\addonPackager\main.py:41
msgid "Empaquetador de complementos"
msgstr "Compilador de extras"

#: addon\globalPlugins\addonPackager\main.py:45
#: addon\globalPlugins\addonPackager\main.py:68
#: addon\globalPlugins\addonPackager\main.py:88
#: addon\globalPlugins\addonPackager\main.py:175
msgid "&Lista de complementos:"
msgstr "&lista de complementos:"

#: addon\globalPlugins\addonPackager\main.py:54
#: addon\globalPlugins\addonPackager\main.py:77
#: addon\globalPlugins\addonPackager\main.py:97
#: addon\globalPlugins\addonPackager\main.py:123
#: addon\globalPlugins\addonPackager\main.py:164
msgid "&Selección"
msgstr "&Selecção"

#: addon\globalPlugins\addonPackager\main.py:57
msgid "&Generar"
msgstr "&Gerar"

#: addon\globalPlugins\addonPackager\main.py:61
msgid "Instalador múltiple"
msgstr "Multi-instalador"

#: addon\globalPlugins\addonPackager\main.py:65
msgid "&Seleccione un directorio con complementos a instalar..."
msgstr "&Selecione um diretório com extras para instalar..."

#: addon\globalPlugins\addonPackager\main.py:80
msgid "&Instalar"
msgstr "&Instalar"

#: addon\globalPlugins\addonPackager\main.py:84
msgid "Desinstala complementos"
msgstr "Desinstalar add-ons"

#: addon\globalPlugins\addonPackager\main.py:100
msgid "&Desinstalar"
msgstr "& Desinstalar"

#: addon\globalPlugins\addonPackager\main.py:104
msgid "Habilita / deshabilita complementos"
msgstr "Activar/desactivar add-ons"

#: addon\globalPlugins\addonPackager\main.py:108
msgid "&Lista de complementos habilitados:"
msgstr "&lista de extras activados:"

#: addon\globalPlugins\addonPackager\main.py:114
msgid "&Lista de complementos deshabilitados:"
msgstr "&Lista de extras desactivados"

#: addon\globalPlugins\addonPackager\main.py:126
#: addon\globalPlugins\addonPackager\main.py:167
msgid "&Procesar"
msgstr "&Processar"

#: addon\globalPlugins\addonPackager\main.py:130
msgid "Modificador de manifiestos"
msgstr "Modificador de manifestos"

#: addon\globalPlugins\addonPackager\main.py:134
msgid "&Lista de complementos instalados:"
msgstr "&lista de add-ons instalados:"

#: addon\globalPlugins\addonPackager\main.py:143
msgid "Seleccione versión Mayor:"
msgstr "Sélectionner la version Majeure:"

#: addon\globalPlugins\addonPackager\main.py:149
msgid "Seleccione versión Menor:"
msgstr "Sélectionner la version Mineure:"

#: addon\globalPlugins\addonPackager\main.py:155
msgid "Seleccione una revisión:"
msgstr "Sélectionner une révision:"

#: addon\globalPlugins\addonPackager\main.py:171
msgid "Documentación de complementos"
msgstr "Documentação adicional"

#: addon\globalPlugins\addonPackager\main.py:181
msgid "&Abrir documentación del complemento"
msgstr "&Abrir documentação do extra"

#: addon\globalPlugins\addonPackager\main.py:187
msgid "&Estado:"
msgstr "&Estado:"

#: addon\globalPlugins\addonPackager\main.py:199
msgid "&Aceptar"
msgstr "&Aceitar"

#: addon\globalPlugins\addonPackager\main.py:202
msgid "&Cancelar"
msgstr "&Cancelar"

#: addon\globalPlugins\addonPackager\main.py:205
msgid "Cerrar"
msgstr "Fechar"

#: addon\globalPlugins\addonPackager\main.py:324
#: addon\globalPlugins\addonPackager\main.py:369
#: addon\globalPlugins\addonPackager\main.py:388
#: addon\globalPlugins\addonPackager\main.py:408
#: addon\globalPlugins\addonPackager\main.py:412
#: addon\globalPlugins\addonPackager\main.py:416
#: addon\globalPlugins\addonPackager\main.py:438
msgid "&Seleccionar todo"
msgstr "&Seleccionar tudo"

#: addon\globalPlugins\addonPackager\main.py:325
#: addon\globalPlugins\addonPackager\main.py:370
#: addon\globalPlugins\addonPackager\main.py:389
#: addon\globalPlugins\addonPackager\main.py:409
#: addon\globalPlugins\addonPackager\main.py:413
#: addon\globalPlugins\addonPackager\main.py:417
#: addon\globalPlugins\addonPackager\main.py:439
msgid "&Deseleccionar todo"
msgstr "&Retirar a selecção"

#: addon\globalPlugins\addonPackager\main.py:331
msgid "Tiene que seleccionar un complemento para continuar."
msgstr "Tem de seleccionar um extra para continuar."

#: addon\globalPlugins\addonPackager\main.py:331
#: addon\globalPlugins\addonPackager\main.py:376
#: addon\globalPlugins\addonPackager\main.py:395
#: addon\globalPlugins\addonPackager\main.py:426
#: addon\globalPlugins\addonPackager\main.py:562
msgid "Error"
msgstr "Erro"

#. Botón selecciona directorio con complementos instalador.
#: addon\globalPlugins\addonPackager\main.py:334
#: addon\globalPlugins\addonPackager\main.py:348
msgid "Seleccione un directorio:"
msgstr "Seleccione um directório:"

#: addon\globalPlugins\addonPackager\main.py:354
msgid "No se encontraron complementos en el directorio que selecciono."
msgstr "Não foram encontrados extras no directório que seleccionou."

#: addon\globalPlugins\addonPackager\main.py:376
#: addon\globalPlugins\addonPackager\main.py:395
#: addon\globalPlugins\addonPackager\main.py:426
#: addon\globalPlugins\addonPackager\main.py:562
msgid "Necesita elegir al menos un complemento para continuar."
msgstr "É necessário escolher pelo menos um complemento para continuar."

#: addon\globalPlugins\addonPackager\main.py:414
msgid "Complementos &habilitados"
msgstr "&lista de extras activados:"

#: addon\globalPlugins\addonPackager\main.py:418
msgid "Complementos &deshabilitados"
msgstr "Complementos &desactivados"

#: addon\globalPlugins\addonPackager\main.py:444
msgid "Procesar &instalados"
msgstr "Processar &instalados"

#: addon\globalPlugins\addonPackager\main.py:445
msgid "Procesar un archivo de &complemento"
msgstr "Processamento de um ficheiro do &extra"

#. Manifiestos procesar fichero.
#: addon\globalPlugins\addonPackager\main.py:576
msgid "Complemento de NVDA (*.nvda-addon)|*.nvda-addon"
msgstr "extra para o NVDA (*.nvda-addon)|*.nvda-addon"

#: addon\globalPlugins\addonPackager\main.py:577
msgid "Seleccione un complemento de NVDA"
msgstr "Seleccione um extra do NVDA"

#: addon\globalPlugins\addonPackager\utilidades.py:111
msgid "Creando el complemento: {} {}"
msgstr "A criar o complemento: {} {}"

#: addon\globalPlugins\addonPackager\utilidades.py:115
msgid ""
"Se generaron correctamente todos los complementos.\n"
"\n"
"Pulse aceptar para continuar."
msgstr ""
"Todos os extras foram criados correctamente.\n"
"\n"
"Pressione aceitar, para continuar."

#: addon\globalPlugins\addonPackager\utilidades.py:121
#: addon\globalPlugins\addonPackager\utilidades.py:203
#: addon\globalPlugins\addonPackager\utilidades.py:253
#: addon\globalPlugins\addonPackager\utilidades.py:385
#: addon\globalPlugins\addonPackager\utilidades.py:439
#: addon\globalPlugins\addonPackager\utilidades.py:502
msgid ""
"Se produjo un error inesperado. \n"
"\n"
"Error: {}\n"
"\n"
"Vuelva a intentarlo."
msgstr ""
"Ocorreu um erro inesperado\n"
"\n"
"Erro: {}\n"
"\n"
"Por favor, tente novamente."

#: addon\globalPlugins\addonPackager\utilidades.py:144
msgid "Instalando el complemento: {}"
msgstr "A instalar o add-on: {}"

#: addon\globalPlugins\addonPackager\utilidades.py:167
msgid ""
"Se completo la instalación correctamente.\n"
"\n"
"NVDA necesita reiniciarse para aplicar las instalaciones.\n"
"\n"
"¿Desea reiniciar NVDA ahora?"
msgstr ""
"A instalação foi concluída com sucesso.\n"
"\n"
"O NVDA precisa de reiniciar para aplicar as instalações.\n"
"\n"
"quer reiniciar o NVDA agora?"

#: addon\globalPlugins\addonPackager\utilidades.py:177
msgid ""
"No se pudo instalar el complemento.\n"
"\n"
"Fallo de compatibilidad.\n"
"\n"
"Busque un complemento compatible."
msgstr ""
"O extra não pôde ser instalado.\n"
"\n"
"Falha de compatibilidade.\n"
"\n"
"Procure um extra compatível."

#: addon\globalPlugins\addonPackager\utilidades.py:188
msgid ""
"Se completo la instalación correctamente.\n"
"\n"
"Pero hay complementos que no se pudieron instalar.\n"
"\n"
"Los siguientes complementos son incompatibles, busque una versión "
"compatible:\n"
"\n"
"{}\n"
"\n"
"NVDA necesita reiniciarse para aplicar las instalaciones.\n"
"¿Desea reiniciar NVDA ahora?"
msgstr ""
"A instalação foi concluída com sucesso.\n"
"\n"
"Mas há extras que não puderam ser instalados.\n"
"\n"
"Os seguintes extras são incompatíveis, por favor procure uma versão "
"compatível:\n"
"\n"
"{}\n"
"\n"
"O NVDA precisa de ser reiniciado para aplicar as instalações.\n"
"Quer reiniciar o NVDA agora?"

#: addon\globalPlugins\addonPackager\utilidades.py:212
msgid ""
"Se produjo un error inesperado. \n"
"\n"
"Error: {}\n"
"\n"
"No obstante hubo complementos que se instalaron y NVDA necesita reiniciar "
"para aplicar los cambios.\n"
"\n"
"¿Desea reiniciar NVDA ahora?"
msgstr ""
"Ocorreu um erro inesperado\n"
"\n"
"Erro: {}\n"
"\n"
"No entanto, alguns extras foram instalados e o NVDA precisa de ser "
"reiniciado para aplicar as alterações.\n"
"\n"
"quer reiniciar o NVDA agora?"

#: addon\globalPlugins\addonPackager\utilidades.py:237
msgid "Desinstalando el complemento: {}"
msgstr "A desinstalar o add-on: {}"

#: addon\globalPlugins\addonPackager\utilidades.py:244
msgid ""
"Se completo la desinstalación correctamente.\n"
"\n"
"NVDA necesita reiniciarse para aplicar los cambios.\n"
"\n"
"¿Desea reiniciar NVDA ahora?"
msgstr ""
"A remoção foi concluída com sucesso.\n"
"\n"
"O NVDA precisa de ser reiniciado para aplicar as alterações.\n"
"\n"
"quer reiniciar o NVDA agora?"

#: addon\globalPlugins\addonPackager\utilidades.py:262
msgid ""
"Se produjo un error inesperado. \n"
"\n"
"Error: {}\n"
"\n"
"No obstante hubo complementos que se desinstalaron y NVDA necesita reiniciar "
"para aplicar los cambios.\n"
"\n"
"¿Desea reiniciar NVDA ahora?"
msgstr ""
"Ocorreu um erro inesperado\n"
"\n"
"Erro: {}\n"
"\n"
"No entanto, houve extras que foram desinstalados e o NVDA precisa de ser "
"reiniciado para aplicar as alterações.\n"
"\n"
"quer reiniciar o NVDA agora?"

#: addon\globalPlugins\addonPackager\utilidades.py:310
msgid "Buscando complementos en {}..."
msgstr "À procura de complementos em {}..."

#: addon\globalPlugins\addonPackager\utilidades.py:314
msgid "No hay complementos compatibles en el directorio."
msgstr "Não existem extras compatíveis no directório."

#: addon\globalPlugins\addonPackager\utilidades.py:318
msgid ""
"Se completo con éxito la extracción de información de los complementos.\n"
"\n"
"No obstante, el directorio que introdujo contiene complementos no "
"compatibles.\n"
"\n"
"* Complementos con fallo en el manifiesto: {}.\n"
"\n"
"{}\n"
"\n"
"* Complementos no compatibles con la api actual: {}.\n"
"\n"
"{}\n"
"\n"
"Pulse aceptar para continuar."
msgstr ""
"A extracção de informação dos extras foi concluída com sucesso.\n"
"\n"
"No entanto, o directório que inseriu contém extras não suportados.\n"
"\n"
"* Plugins com falhas manifestas: {}.\n"
"\n"
"{}\n"
"\n"
"* Plugins não compatíveis com a api actual: {}.\n"
"\n"
"{}\n"
"\n"
"Pressione aceito para continuar."

#: addon\globalPlugins\addonPackager\utilidades.py:334
msgid ""
"Se completo con éxito la extracción de información de los complementos.\n"
"\n"
"Pulse aceptar para continuar."
msgstr ""
"A extracção de informação dos add-ons foi concluída com sucesso.\n"
"\n"
"Clique OK para continuar."

#: addon\globalPlugins\addonPackager\utilidades.py:357
msgid "Deshabilitando complementos..."
msgstr "A desactivar add-ons..."

#: addon\globalPlugins\addonPackager\utilidades.py:367
msgid "Habilitando complementos..."
msgstr "A activar os extras."

#: addon\globalPlugins\addonPackager\utilidades.py:376
msgid ""
"Se completo el proceso de habilitar y deshabilitar correctamente.\n"
"\n"
"NVDA necesita reiniciarse para aplicar los cambios.\n"
"\n"
"¿Desea reiniciar NVDA ahora?"
msgstr ""
"O processo de activação e desactivação foi concluído com êxito.\n"
"\n"
"O NVDA precisa de ser reiniciado para aplicar as alterações.\n"
"\n"
"quer reiniciar o NVDA agora?"

#: addon\globalPlugins\addonPackager\utilidades.py:394
msgid ""
"Se produjo un error inesperado. \n"
"\n"
"Error: {}\n"
"\n"
"No obstante hubo complementos que se habilitaron o deshabilitaron y NVDA "
"necesita reiniciar para aplicar los cambios.\n"
"\n"
"¿Desea reiniciar NVDA ahora?"
msgstr ""
"Ocorreu um erro inesperado\n"
"\n"
"Erro: {}\n"
"\n"
"No entanto, alguns extras foram activados ou desactivados e o NVDA precisa "
"de ser reiniciado para aplicar as alterações.\n"
"\n"
"quer reiniciar o NVDA agora?"

#: addon\globalPlugins\addonPackager\utilidades.py:423
msgid "Cambiando el manifiesto al complemento: {}"
msgstr "A alterar o manifesto para o complemento: {}"

#: addon\globalPlugins\addonPackager\utilidades.py:430
msgid ""
"Se cambiaron los manifiestos correctamente.\n"
"\n"
"Para que la acción tenga efecto NVDA tiene que reiniciarse.\n"
"\n"
"¿Desea reiniciar NVDA ahora?"
msgstr ""
"Os ficheiros manifest.ini foram alterados correctamente.\n"
"\n"
"Para que a acção tenha efeito, o NVDA tem de ser reiniciado.\n"
"\n"
"Quer reiniciar o NVDA agora?"

#: addon\globalPlugins\addonPackager\utilidades.py:448
msgid ""
"Se produjo un error inesperado. \n"
"\n"
"Error: {}\n"
"\n"
"No obstante hubo complementos que se les cambio el manifiesto y NVDA "
"necesita reiniciar para aplicar los cambios.\n"
"\n"
"¿Desea reiniciar NVDA ahora?"
msgstr ""
"Ocorreu um erro inesperado\n"
"\n"
"Erro: {}\n"
"\n"
"Contudo, houve extras que tiveram o seu manifesto alterado e o NVDA precisa "
"de ser reiniciado para aplicar as alterações.\n"
"\n"
"quer reiniciar o NVDA agora?"

#: addon\globalPlugins\addonPackager\utilidades.py:477
msgid "Trabajando en el complemento: {}"
msgstr "A trabalhar no extra: {}"

#: addon\globalPlugins\addonPackager\utilidades.py:492
msgid ""
"Se cambio correctamente el manifiesto.\n"
"\n"
"Se genero un complemento con el manifiesto cambiado en:\n"
"\n"
"{}\n"
"\n"
"Pulse aceptar para continuar."
msgstr ""
"O manifesto foi correctamente alterado.\n"
"\n"
"Foi criado um extra com o manifesto alterado:\n"
"\n"
"{}\n"
"\n"
"Clique OK para continuar."
